tests/utils/isReqFrom.test.js

```diff
@@ -0,0 +1,77 @@
+
+
+jest.mock('dns', () => ({
+  promises: {
+    reverse: jest.fn((ip) => {
+      if (ip === '127.0.0.1') return [];
+      return ['ip-192-168-905-123.region.compute.internal'];
+    }),
+  },
+}));
+
+const {
+  isReqFromCluster,
+  isReqFromLocalhost,
+} = require('../../src/utils/isReqFrom');
+const fake = require('../test-utils/constants');
+
+describe('Tests for isRequestFromCluster and isRequestFromLocalhost', () => {
```

Comment:
you could have also one `describe('isRequestFromCluster')` and another `describe('isRequestFromLocalhost')` (in the same `.test.js` file). 

It's not a PR-blocking thing though, but just leaving this as another option.